src/object/transform_cl.c

```diff
@@ -2158,12 +2166,34 @@ disk_to_domain2 (OR_BUF * buf)
       return NULL;
     }
 
+  if (vars[ORC_DOMAIN_SCHEMA_JSON_OFFSET].length > 0)
+    {
+      char *schema_raw;
+      int error_code;
+
+      or_get_json_schema (buf, schema_raw);
+      if (schema_raw == NULL)
+	{
+	  er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_OUT_OF_VIRTUAL_MEMORY, 1,
+		  vars[ORC_DOMAIN_SCHEMA_JSON_OFFSET].length + 1);
+	  tp_domain_free (domain);
+	  return NULL;
```

Comment:
`vars` might be leaked. 
It seems it is better to have an error handler.
please also notice that `domain` might be leaked at 2166.